cmd/ike/cmd/root.go

```diff
@@ -28,18 +38,65 @@ Duis nec interdum velit, id consectetur erat. In tempor tempor turpis vel rhoncu
 }
 
 func startOperator() {
-	sdk.ExposeMetricsPort()
-	resource := "istio.openshift.com/v1alpha1"
-	kind := "Session"
 	namespace, err := k8sutil.GetWatchNamespace()
 	if err != nil {
-		logrus.Fatalf("Failed to get watch namespace: %v", err)
+		log.Error(err, "Failed to get watch namespace")
+		os.Exit(1)
+	}
+
+	// Get a config to talk to the apiserver
+	cfg, err := config.GetConfig()
+	if err != nil {
+		log.Error(err, "")
```

Comment:
Are you moving away from `logrus` and switching to `log`?

Reply:
logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"

https://github.com/aslakknutsen/istio-workspace/pull/6/files#diff-b9de94200d7321a9f4eb6309638db08cR13